hs_core/views/__init__.py

```diff
@@ -619,6 +619,17 @@ def delete_metadata_element(request, shortkey, element_name, element_id, *args,
     return HttpResponseRedirect(request.META['HTTP_REFERER'])
 
 
+def delete_author(request, shortkey, element_id, *args, **kwargs):
+    try:
+        res, _, _ = authorize(request, shortkey, needed_permission=ACTION_TO_AUTHORIZE.EDIT_RESOURCE)
+        res.metadata.delete_element('creator', element_id)
+        resource_modified(res, request.user, overwrite_bag=False)
+        ajax_response_data = {'status': 'success', 'message': "Author was deleted successfully"}
+    except Error as exp:
```

Comment:
With try/catch it's best to be specific.  Keep code in the try block small and be specific about the error.  What errors are you trying to catch here?

Reply:
Understood. In this case I was trying to catch database errors.